docs/stripe-tests.md

```diff
@@ -74,6 +74,20 @@ In order to run, no matter if it is in test mode or production mode, the Stripe
 
 That, in turn, means that tests against the Stripe SDK will crash unless those tests are run against an app. So to workaround this, we moved the integration tests to the WooCommerceTests suite. See `StripeCardReaderIntegrationTests`
 
+Integration tests that go just beyond discovering readers are not easy to run (at this stage). The reason is that the Terminal SDK needs a valid connection token when it attempts to connect to a reader, even if the SDK is in test mode.
+
+There are some prerequistes for running some of the integration tests:
+
+* Install WooCommerce Payments on a WooCommerce site. Do not run the setup assisntant yet.
+* Install [the dev plugin](https://github.com/Automattic/woocommerce-payments-dev-tools) by downloading the repo as a zip and uploading it to the site as a new plugin, and activate it.
```

Comment:
This (woocommerce-ios) repo is public. Should we refer to a private repo? As an alternative, the end user could add this to their wp-config.php or somewhere.

```php
define( 'WCPAY_DEV_MODE', true );
```

Reply:
I think for now, until we extract networking and implement the request to fetch the token properly, it should be fine? I'll ask